packages/vite-plugin-cloudflare/playground/vitest-setup.ts

```diff
@@ -77,13 +76,13 @@ export function setViteUrl(url: string): void {
 }
 
 export function resetServerLogs() {
-	serverLogs.info = [];
-	serverLogs.warns = [];
-	serverLogs.errors = [];
+	serverLogs.info.splice(0, serverLogs.info.length);
+	serverLogs.warns.splice(0, serverLogs.warns.length);
+	serverLogs.errors.splice(0, serverLogs.errors.length);
```

Comment:
What was the reason for this change?

Reply:
This allows us resetting the server logs in the middle of a tests. The previous solution assigns a new array but the in-memory logger is still referencing the old array. So once you reset it, you won't be able to see any new logs.